test/entrypoint/srv_batch_operations/cartridge_init.lua

```diff
@@ -11,11 +11,11 @@ local cartridge = require('cartridge')
 if package.setsearchroot ~= nil then
     package.setsearchroot()
 else
-    package.path = package.path .. debug.sourcedir() .. "/?.lua;"
+    package.path = package.path .. debug.sourcedir() .. "/?.lua;" .. debug.sourcedir() .. "/?/init.lua;"
```

Comment:
Nit: please, let's describe the motivation in the commit msg and the gh ticket number, if it's applicable

Reply:
Added description to the commit message, there are no related tickets though.